cylc/uiserver/authorise.py

```diff
@@ -160,9 +159,8 @@ def set_auth_conf(auth_conf: Union[LazyConfigValue, dict]) -> dict:
         Returns:
             Valid configuration dictionary
         """
-        if isinstance(auth_conf, LazyConfigValue) and not auth_conf.to_dict():
-            # no owner user auth - return empty dict
-            return {}
+        if isinstance(auth_conf, LazyConfigValue):
+            return auth_conf.to_dict()
         return auth_conf
```

Comment:
If `auth_conf.to_dict()` is falsey it must be an empty dict anyway.

I have changed the logic here so this function always returns a dict.